src/main/webapp/front-end/src/utilities.js

```diff
@@ -1,35 +1,43 @@
 import store from './store.js';
+import Cookies from 'js-cookie';
 import { login, loginError, loginSuccess, loginClearInputs } from './actions.js';
 var _ = require('lodash');
 
-export var authenticatedSession = null;
+export var authenticationHeader = 'bee-session-key'
+export var cookieName = 'BEE_SESS_KEY'
```

Comment:
i'm no expert, but seems purely confusing to have different names for these keys.  any thoughts?